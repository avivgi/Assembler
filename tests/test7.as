.define sz = 2
K: .data 22
LIST: .data 6, 8, sz

; zero operands
rts

END: hlt