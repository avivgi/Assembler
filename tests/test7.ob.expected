4 4
0100 00000000100100 ****%#*
0101 00000110100110 **#%%#%
0102 00000000001000 *****%*
0103 00000110100010 **#%%*%
0104 11111111101010 !!!!%%%
0105 00000000000110 *****#%
0106 00000000001000 *****%*
0107 11111111111110 !!!!!!%
